src/utils/interaction.py

```diff
@@ -38,6 +38,9 @@ def show(name, origin, pause=1, resize=False, reset_pos=None, config=None):
         else:
             img = origin
 
+        if not is_window_available(name):
+            cv2.namedWindow(name)
```

Comment:
can you add this for the cv2.moveWindow() as well?

Reply:
Its not needed separately for the same, We can do that but it does not affect anything as we are already handling the case when we don't have window here, so no need to do it again.